source/solvers/tracer_assemblers.cc

```diff
@@ -345,3 +446,256 @@ TracerAssemblerBDF<dim>::assemble_rhs(TracerScratchData<dim>    &scratch_data,
 
 template class TracerAssemblerBDF<2>;
 template class TracerAssemblerBDF<3>;
+
+
+
+template <int dim>
+void
+TracerAssemblerSIPG<dim>::assemble_matrix(
+  const TracerScratchData<dim> &scratch_data,
+  StabilizedDGMethodsCopyData  &copy_data)
```

Comment:
Good question. I did not want to create a CopyData that was not stabilized for DG because initially I thought we would need stabilization. Since we don't, I can remove that.

Reply:
Ohh I know why. If we do not make it inherit of the regular stabilized method copy data then we cannot use the regular cell assembly functions. So consequently, it means that we would need to rewrite the loop over local cells. This is an improvement that could be done in the future, but right now this has no cost and really reduces code duplication TBH.